### ISSUE TITLE
Can not use atob function because of different built-in libraries

### ISSUE THREAD

**Issue by fourcolor** [apideck-libraries/postman-to-k6#86]
I use many atob function in my postman collection; indeed, it not work in k6 script. However I found that I can easily fix it by using the k6/encoding module.
```javascript
import { b64decode } from "k6/encoding";
const atob = (str) => {
  return b64decode(str, "rawstd", "s");
};
```
I want to ask is there any plan to support this kind of problem?
e.g. crypto-js -> k6/crypto

**Comment by thim81** [comment]
hi @fourcolor,

Do I understand correctly, that you want to add the atob() function when it is detected in the Postman collection?

**Comment by fourcolor** [comment]
1. Yes
2. Here is the example
```javascript=
pm.test("Living JWT token", function () {
    let jsonData = pm.response.json();
    let Playload = JSON.parse(atob(jsonData.JWT.split(".")[1]));
    let iat = new Number(Playload.iat);
    let exp = new Number(Playload.exp);
    let dateToSecond = Math.ceil(Date.now()/1000);
    //check JWT token was issued in the past and expire in the future which means it is an effect token for now
    pm.expect(dateToSecond).to.be.within(iat,exp);
});
```

**Comment by thim81** [comment]
hi @fourcolor 

Picking this up again.

So your postman post-request script looks like this:

```
pm.test("Living JWT token", function () {
    let jsonData = pm.response.json();
    let Playload = JSON.parse(atob(jsonData.JWT.split(".")[1]));
    let iat = new Number(Playload.iat);
    let exp = new Number(Playload.exp);
    let dateToSecond = Math.ceil(Date.now()/1000);
    //check JWT token was issued in the past and expire in the future which means it is an effect token for now
    pm.expect(dateToSecond).to.be.within(iat,exp);
});
```

How do your work-around K6 generated script looks like?

Based on your input, we create some additional logic to use the k6/encoding instead of the crypto-js ATOB.

**Comment by thim81** [comment]
hi @fourcolor 

We just released version 1.13.0 which contains an atob implementation using the K6 provided b64decode function.
More details: https://github.com/apideck-libraries/postman-to-k6/blob/main/lib/shim/atob.js
